labelbox/schema/project.py

```diff
@@ -1454,12 +1454,35 @@ def _wait_until_data_rows_are_processed(
         """ Wait until all the specified data rows are processed"""
         start_time = datetime.now()
```

Comment:
Since the start_time is set to the beginning of this function, I think `_poll_data_row_processing_status` may not wait the full `wait_processing_max_seconds` per chunk

Reply:
My understanding was that the `wait_processing_max_seconds` was to be applied to the entire call for `_wait_until_data_rows_are_processed`, and not the individual chunks. This should preserve original behavior

Reply:
Ah gotcha makes sense